src/applications/personalization/profile/sass/profile.scss

```diff
@@ -63,18 +63,22 @@
   overflow: auto;
 }
 
+.va-profile-alert.usa-alert-error::before {
```

Comment:
we should be able to do all of this in the JSX, using existing utility classes and `fa-` classes without needing to add new CSS, similar to how the `IconCTALink` is set up https://github.com/department-of-veterans-affairs/vets-website/blob/8736710ac1de0b5c7a780487bba9c5ca4216b470/src/applications/personalization/dashboard/components/IconCTALink.jsx#L33-L53

The markup you'd need would actually be simpler than that, though, since you only need a single `i` element for the icon instead of stacking two on top of each other.

Reply:
Nice! We could do that. I would then be going back to not using the Alertbox component I believe.

Reply:
Just pushed new updates, let me know what you think.
![Screen Shot 2021-12-01 at 2 41 46 PM](https://user-images.githubusercontent.com/26075258/144303741-4d2a43f0-e0e3-42a5-8330-eceaa62c5ab1.png)